src/shared/telemetry/activation.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { AwsContext } from '../awsContext'
+import { SettingsConfiguration } from '../settingsConfiguration'
+import { DefaultTelemetryService } from './defaultTelemetryService'
+import { ext } from '../extensionGlobals'
+import { getLogger } from '../logger'
+import { TelemetryService } from './telemetryService'
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+// SETTINGS_TELEMETRY_VALUE_XXX must be kept in sync with package.json "aws.telemetry" configuration values
+const SETTINGS_TELEMETRY_VALUE_ENABLE = 'Enable'
+const SETTINGS_TELEMETRY_VALUE_DISABLE = 'Disable'
+const SETTINGS_TELEMETRY_VALUE_USEIDE = 'Use IDE settings'
```

Comment:
Can those and these use localized strings (if we're concerned about language, maybe we can make sure those aren't ever translated)?

Reply:
These are the values saved to settings. They shouldn't be localized. We're ensuring they're never translated by using constants.

These were moved, there isn't anything new about these constants in this change.